contracts/axelarnet-gateway/src/clients/gateway.rs

```diff
@@ -61,10 +55,8 @@ impl Client<'_> {
             .map(|messages| self.client.execute(&ExecuteMsg::RouteMessages(messages)))
     }
 
-    pub fn chain_name(&self) -> Result<ChainName, Error> {
-        self.client
-            .query(&QueryMsg::ChainName)
-            .change_context_lazy(|| Error::QueryChainName(self.client.address.clone()))
+    pub fn chain_name(&self) -> Result<ChainName, client::Error> {
+        self.client.query(&QueryMsg::ChainName)
```

Comment:
this required additional exposed data, and is redundant anyway, because the same error is thrown further out